catalog/transformer/catalog-transformer-pptx/pom.xml

```diff
@@ -0,0 +1,250 @@
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <version>1.4</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>compile</goal>
+                            <goal>testCompile</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.name}</Bundle-Name>
+                        <Import-Package>
+                            !com.ibm.icu.text,
+                            !org.etsi.uri.x01903.v14.*,
+                            ddf.catalog.*,
+                            org.slf4j.*,
+                            com.google.common.io.*,
+                            org.xml.sax.*,
+                            org.w3c.dom.*,
+                            javax.xml.*,
+                            javax.imageio.*,
+                            org.apache.commons.logging.*
+                        </Import-Package>
+                        <Embed-Dependency>
+                            catalog-core-api-impl,
+                            platform-util,
+                            catalog-transformer-common,
+
+                            <!-- external -->
+
+                            bcprov-jdk15on,
+                            bcpkix-jdk15on,
+                            <!--
+
+                            Big nasty hack. Should not normally embed another bundle. But we
+                            can't install this bundle separately in osgi because of a missing
+                            class in the bundle. So far, I've sent an email to
+                            users@servicemix.apache.org asking for input, and they created a
+                            ticket for it. When the next servicemix release is made, we should
+                            check to see if the error has been fixed, and update accordingly.
+
+                            This is the apache servicemix ticket:
+
+                            https://issues.apache.org/jira/browse/SM-2925
+
+                            -->
+                            org.apache.servicemix.bundles.poi,
```

Comment:
I'd rather see us embed the poi artifacts directly instead of embedding this bundle.